include/wx/cmndata.h

```diff
@@ -145,6 +145,24 @@ class WXDLLIMPEXP_CORE wxPrintData: public wxObject
     wxDECLARE_DYNAMIC_CLASS(wxPrintData);
 };
 
+/*
+ * wxPrintPageRange
+ * Contains a range of pages to be printed.
+ */
+
+class WXDLLIMPEXP_CORE wxPrintPageRange
+{
+public:
+    int fromPage = 0;
+    int toPage = 0;
+
+    wxPrintPageRange() : fromPage(0), toPage(0) { }
+    wxPrintPageRange(int _fromPage, int _toPage) : fromPage(_fromPage), toPage(_toPage) { }
```

Comment:
Arguably this is not needed neither, we should be able to initialize this class just by using `{from, to}` as it's a simple aggregate, so I'd consider removing both ctors.

Reply:
I actially like ctors like `wxPrintPageRange(int _fromPage, int _toPage)` because it's easier for programming. Programmers can see more easily in the IDE how to create the object. If you do it with `{from, to}`, you always have to check the order of the elements beforehand and this is more error-prone.

Reply:
Note that you can, of course, write `wxPrintPageRange{from, to}` explicitly too.

`wxPoint` was written for pre-C++98 compilers, things were a bit different back then...

Reply:
I tried without ctors but had build issues on mac, so switched back to this version. You can decide vadz, I'm fine with either version.